sinker/sinker.go

```diff
@@ -210,18 +210,18 @@ func (s *SQLSinker) HandleBlockUndoSignal(ctx context.Context, data *pbsubstream
 	return s.loader.Revert(ctx, s.OutputModuleHash(), cursor, data.LastValidBlock.Number)
 }
 
-func (s *SQLSinker) batchBlockModulo(blockData *pbsubstreamsrpc.BlockScopedData, isLive *bool) uint64 {
+func (s *SQLSinker) batchBlockModulo(isLive *bool) uint64 {
 	if isLive == nil {
 		panic(fmt.Errorf("liveness checker has been disabled on the Sinker instance, this is invalid in the context of 'substreams-sink-sql'"))
 	}
 
 	if *isLive {
-		return LIVE_BLOCK_FLUSH_EACH
+		return uint64(s.loader.LiveBlockFlushInterval())
 	}
 
-	if s.loader.FlushInterval() > 0 {
-		return uint64(s.loader.FlushInterval())
+	if s.loader.BatchBlockFlushInterval() > 0 {
+		return uint64(s.loader.BatchBlockFlushInterval())
 	}
 
-	return HISTORICAL_BLOCK_FLUSH_EACH
+	return 0
```

Comment:
We should probably add a constant `BLOCK_FLUSH_INTERVAL_DISABLED` or something in that vein to more clearly convey the fact that this disables the interval.